OSPSuite.Presentation.Importer/Core/DataFormat/DataFormatParameter.cs

```diff
@@ -12,71 +11,82 @@ public enum DataFormatParameterType
          MetaData
       }
 
-      public ParameterConfiguration(DataFormatParameterType type, object data = null)
+      public DataFormatParameterType Type { get; }
+
+      public ParameterConfiguration(DataFormatParameterType type)
       {
          Type = type;
-         Data = data;
       }
 
-      public DataFormatParameterType Type { get; private set; }
+      public override string ToString()
+      {
+         return $"{Type}";
+      }
+   }
 
-      public object Data { get; private set; }
+   public class ParameterConfiguration<T> : ParameterConfiguration where T : class
```

Comment:
using generic. Not sure how this is going to be  used at the end but wanted to show how it couldwork